keyboards/gingham/rules.mk

```
@@ -0,0 +1,96 @@
+SRC = matrix.c \
+      ../../drivers/avr/i2c_master.c
```

You should be fine using just: 

```suggestion
      i2c_master.c
```